packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -68,6 +126,8 @@ export class Accelerator extends cdk.Resource implements IAccelerator {
     class Import extends cdk.Resource implements IAccelerator {
       public readonly acceleratorArn = attrs.acceleratorArn;
       public readonly dnsName = attrs.dnsName;
+      public readonly ipAddresses = attrs.ipAddresses?? undefined;
+      public readonly ipAddressType = attrs.ipAddressType ?? undefined;
```

Comment:
I don't think the null coalescing operators are necessary since they are originally union types with undefined.

```suggestion
      public readonly ipAddresses = attrs.ipAddresses;
      public readonly ipAddressType = attrs.ipAddressType;
```

Reply:
Depending on [this comment](https://github.com/aws/aws-cdk/pull/28055/files#r1400194833), it will be removed.